Symfony/CS/Fixer/Symfony/ExtraEmptyLinesFixer.php

```diff
@@ -16,10 +16,67 @@
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author SpacePossum <possumfromspace@gmail.com>
  */
 final class ExtraEmptyLinesFixer extends AbstractFixer
 {
     /**
+     * @var string[]|null
+     */
+    private $configuration = array(T_WHITESPACE => 'removeMultipleBlankLines');
+
+    /**
+     * @var Tokens
+     */
+    private $tokens;
+
+    /**
+     * Set configuration.
+     *
+     * Valid configuration option are:
+     * - 'break' remove blank lines after a line with a 'break' statement
+     * - 'continue' remove blank lines after a line with a 'break' statement
+     * - 'extra' [default] remove extra blank lines
+     * - 'return' remove blank lines after a line with a 'break' statement
+     * - 'throw' remove blank lines after a line with a 'break' statement
+     * - 'use' remove blank lines between 'use' import statements
+     *
+     * @param string[]|null $configuration
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            return;
+        }
+
+        $this->configuration = array();
+        foreach ($configuration as $item) {
+            switch ($item) {
+                case 'break':
+                    $this->configuration[T_BREAK] = 'removeEmptyLinesAfterLineWithTokenAt';
```

Comment:
no no no, completly not that, you just change the name and stop using configuration.

have some array like
```php
private $tokenToCallback = [ T_BREAK => 'remove', T_USE => 'sth',  ];
```

maybe it will need to be 2-level, like `[ break => [token=>T_BREAK, callback => 'remove'], ...]`

and don't touch it, don't mess with it, keep it static and read only.

Then, in `configure` method just to `$this->configuration = $configuration` and in method that fix the code look into configuration and tokenToCallback.